enterprise_extensions/frequentist/Fe_statistic.py

```diff
@@ -15,49 +30,23 @@ class FeStat(object):
 
     """
 
-    def __init__(self, psrs, params=None):
+    def __init__(self, psrs, params=None, custom_models={}, inc_crn=False, orf=None, pta=None):
```

Comment:
It looks like the psrs aren't used anymore to build the pta object. That doesn't look like it's compatible with how things were. Is this really necessary? Can we not just allow both ways of creating the FeStat object? This is not backward compatible

Reply:
Yes, totally possible. The new pull request has a more versatile if-statement